llvm/lib/Transforms/Vectorize/VPlan.h

```diff
@@ -865,6 +865,7 @@ class VPInstruction : public VPRecipeWithIRFlags,
     CalculateTripCountMinusVF,
     // Increment the canonical IV separately for each unrolled part.
     CanonicalIVIncrementForPart,
+    WideIVStep,
```

Comment:
Nit, add a doc comment explaining the operands?

Reply:
Thanks, also moved it to the end, let's keep all VPInstructionWithType opcodes there.